aten/src/ATen/native/cpu/IndexKernel.cpp

```diff
@@ -304,15 +304,12 @@ void index_copy_kernel(
 
 template <typename scalar_t, typename mask_t>
```

Comment:
Nit, no need to pass `mask_t` as it is fixed
```suggestion
template <typename scalar_t>
```